pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -283,9 +288,21 @@ func deploymentOnPVC(c *Cluster, osd OSDInfo, pvcName string, config *provisionC
 		return nil, errors.Wrapf(err, "failed to generate deployment for %s", osdLongName)
 	}
 
+	err = setOSDProperties(c, osdProps, osd)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to prepare for deployment %s", osdLongName)
+	}
+
 	return d, nil
 }
 
+func setOSDProperties(c *Cluster, osdProps osdProperties, osd OSDInfo) error {
```

Comment:
Please add a comment that these properties are only added because they could not be set with cmd line args

Reply:
OK